doctr/models/_utils.py

```diff
@@ -88,3 +88,61 @@ def extract_rcrops(img: Union[np.ndarray, tf.Tensor], boxes: np.ndarray) -> List
         crops.append(crop)
 
     return crops
+
+
+def rotate_page(image: np.array, angle: float = 0., min_angle: float = 1.) -> np.array:
+    """Rotate an image counterclockwise by an ange alpha
+
+    Args:
+        alpha: angle, degrees, between -90 and +90
+        image: np array to rotate
+        min_angle: min. angle in degrees to rotate a page
+
+    Returns:
+        Rotated np array
+    """
+    if abs(angle) < min_angle or abs(angle) > 90 - min_angle:
+        return image
+    height, width = image.shape[:2]
+    center = tuple(np.array([height, width]) / 2)
+    rot_mat = cv2.getRotationMatrix2D(center, angle, 1.0)
+    rotated = cv2.warpAffine(image, rot_mat, (width, height))
+    return rotated
+
+
+def get_bitmap_angle(bitmap: np.array, n_ct: int = 5, std_max: float = 3.) -> float:
```

Comment:
I suggest switching to a function taking the rotated bounding boxes as inputs so that we can decorrelate box computation from orientation regression

Reply:
I am not sure to understand your suggestion, I purposely made this function to avoid computing all rotated boxes when we will compute the angle and rotate the page, because we won't need rotated boxes after these steps (but straight boxes).

Reply:
Since we find all the contours in there, I guess it's almost as if we were to compute the rotated boxes. So I figured that we could use this angle computation only if we are to compute rotated boxes (and thus passing them as args of the function)